spec/config_setup_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'spec_helper'
```

Comment:
Tests appear to be passing, but this line is actually failing on CI.
Seems that uSpec needs to better indicate failure states when loading tests.

Reply:
This was fixed in https://github.com/acook/uspec/commit/6c616528295d90b3205087124b86f742768e7f5e and https://github.com/acook/uspec/commit/5a94e66991d411586180fa348fd219eb4e20ae37.

Reply:
Fix deployed in [v0.1.1](https://rubygems.org/gems/uspec/versions/0.1.1).